docs/serving/feature-flags.md

```diff
@@ -275,6 +275,30 @@ spec:
         ...
 ```
 
+## Kubernetes Security Context Capabilities
+
+* **Type**: extension
+* **ConfigMap key**: `kubernetes.podspec-addcapabilities`
+
+This flag controls whether users can add capabilities on the Pod's securityContext.
+
+When set to "enabled" or "allowed" it allows [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html) to be added to the container.
```

Comment:
```suggestion
When set to `enabled` or `allowed` it allows [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html) to be added to the container.
```

Is enabled / allowed a literal setting? Where does someone switch this on/off?

Reply:
This is set in the [`config-features` configmap](https://github.com/psschwei/serving/blob/add-add-capability/config/core/configmaps/features.yaml#L112) in the `knative-serving` namespace. Definitions for the enabled / allowed values are provided in the [`docs/feature-flags.md`](https://github.com/knative/docs/blob/mkdocs/docs/serving/feature-flags.md#control) page.

On `enabled` vs. "enabled" -- happy to make that change. There's a number of other places in the doc where the values are in quotes, would it be better to update them all at once in a separate PR (maybe together with the yaml examples mentioned in a previous comment)?

Reply:
> This is set in the config-features configmap in the knative-serving namespace. Definitions for the enabled / allowed values are provided in the docs/feature-flags.md page.

I'd at least call that out and link to that page maybe, as someone reading this I wouldn't know how to enable it in the configmap without the pointer.

Reply:
Sorry, I got the link wrong... those definitions are actually on this same page (in the first section).